docs/development.md

```diff
@@ -44,6 +44,17 @@ Cleartext is never stored in anything other than memory (and event then it will
 When we are working with a "password", it should always be a [SHA-256](https://en.wikipedia.org/wiki/SHA-2) hash of the cleartext.
 Any variables/arguments actually using cleartext should make that apparent with the variable name (i.e. the name should have "cleartext" in it).
 
+### Test Users
```

Comment:
This information is fine (and we can leave it in),
but it is not what we are looking for here.
You need to describe the semantics of how a server admin is treated when turned into a course user.